### DISCUSSION TITLE
Take off issue

### DISCUSSION THREAD

**Discussion by CloudSurfer111** [bitcraze/discussions#1715] [Q&A]
So I put together my drone, my drone has the propellers showing the indent ring facing up. When I plug it into the battery, the drones rear LED’s show a constant blue light, the front right LED shows a red light that flashes twice per second. Whenever I try to take off (I’m using mobile on Apple), my drone can’t even though the thrust is all the way. And when I try to have the thrust all the way up, the propellers don’t seem to all go at the same time if that makes sense, like I’ll hold it for three seconds, and all propellers will run except for one, and then all 4 will run again.

**Comment by ToveRumar** [comment]
Hi @CloudSurfer111 !

This sounds like it might be misconfigured or something similar. 

Could you either take a photo/video of it when you try to fly it 
or do the following; 

Install the cfclient by calling
`pip install cfclient` 
and launching it by calling `cfclient`.

Connect to your crazyflie via radio or usb, and open up the tab called paramteters. There is a paramteter that is called "motorPowerSet", expand this and set enable setting to 1. Then you can spin the motors one by one to see that they are all spinning normally. 

This will give some more info about the nature of the issue.
